codegen/README.md

```diff
@@ -1,4 +1,4 @@
-# gson-codegen
```

Comment:
Was an oversight by me; all other modules are using the folder name (instead of the artifact ID).